### PR TITLE
Add GitHub Token leak process

### PR DESCRIPTION
**Pull request by tabbysable**
Add a lightweight guide for responding to a GitHub token leak.

### CODE REVIEW

File: token-leak-process.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+* Thank the reporter -- they have done us a kindness by letting us know about the problem
+* If you can, get the first few characters of the leaked token in case GitHub support needs it for revocation
+* Check to see if the user is an OWNER - an OWNER token leak would be more severe than that of an ordinary Org member
+    * https://cs.k8s.io/?i=fosho&files=OWNERS&q=the-user-name-goes-here
+
+## Contact
+
+* Contact the user via email, letting them know about the token leak
+>Dear USER,
```
Comment (tallclair) [inline token-leak-process.md:15]: nit: consider bolding or using code blocks to make the things that need replacing pop more:

>Dear **USER**,

or

>Dear `USER`,

**Comment by tallclair** [comment]
Can we create a `playbook` directory, and add this there?

File: token-leak-process.md (hunk 1 above)
Comment (tabbysable) [reply token-leak-process.md]: Done!

File: playbook/token-leak-process.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+## Investigate
+
+* Thank the reporter -- they have done us a kindness by letting us know about the problem
+* If you can, get the first few characters of the leaked token in case GitHub support needs it for revocation
+* Check to see if the user is an OWNER - an OWNER token leak would be more severe than that of an ordinary Org member
+    * https://cs.k8s.io/?i=fosho&files=OWNERS&q=the-user-name-goes-here
+
+## Contact
+
+* Contact the user via email, letting them know about the token leak
```
Comment (ritazh) [inline playbook/token-leak-process.md:14]: Which email should we use to send this?

File: playbook/token-leak-process.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+>We were recently notified that a personal GitHub token has been leaked (say a little about how the token was leaked, if possible. e.g. "via CI logs visible on https://example.net/ci-logs").
+>
+>Please revoke the leaked GitHub token, to protect yourself and your projects. Let us know if you need any further information.
+>
+>Best,
+>
+>`YOUR NAME HERE`  
+>Kubernetes SRC
+* If the user is an OWNER or might have other privileged roles in the project, contact a ContribEx chair or Steering to discuss whether further incident response is needed
+* Contact a GitHub admin to submit a support ticket asking for the token to be revoked. They should cc our contacts there, who will escalate with the term “3rd party revocation”
```
Comment (ritazh) [inline playbook/token-leak-process.md:26]: Is this our Github org admins or something else?

Comment (enj) [reply playbook/token-leak-process.md:26]: This refers to the back channel IIUC.  Can we add a link on how to do this?

File: playbook/token-leak-process.md (hunk 2 above)
Comment (enj) [reply playbook/token-leak-process.md:14]: Probably whatever is listed on their GH account / commits.

Comment (tabbysable) [reply playbook/token-leak-process.md]: Added a little detail

File: playbook/token-leak-process.md (hunk 3 above)
Comment (tabbysable) [reply playbook/token-leak-process.md]: Added a little detail

**Comment by SaranBalaji90** [comment]
@tabbysable thank you for the PR. I approved it, looking at the current revision it looks good to me and don't want the bot to close the PR again.

**Pull request merged**